### PR TITLE
Add 6 social links

### PR DESCRIPTION
**Pull request by kubami9**
**Add 6 github links for:**
 - Hong Phuc Dang
 - Mario Behling
 - Harish Pillay
 - Roland Turner
 - Darwin Gosal
 - Archy Wilhes

### CODE REVIEW

**Review by isuruAb — APPROVED** [review]

**Comment by isuruAb** [comment]
@kubami9  please provide a demo link

**Comment by kubami9** [comment]
@isuruAb I made this link: https://kubami9.github.io/2015.fossasia.org-km-contribute/, however it doesn't show any changes and I don't know why since index.html is changed 🤔

**Review by akshatnitd — COMMENTED** [review]
You need to change the github icon class name, since this class doesn't show the icons!

**Review by akshatnitd — CHANGES_REQUESTED** [review]
Try to add a github icon, the one you added is for external links

File: index.html [hunk 1]
```diff
@@ -542,6 +542,9 @@ <h1>Organizers</h1>
 												<a href="https://www.linkedin.com/in/hongphucdang" target="_self">
 													<i class="icon social_linkedin"></i>
 												</a>
+												<a href="https://github.com/hpdang" target="_self">
+													<i class="icon icon_link_alt"></i>
```
Comment (abishekvashok) [inline index.html]: You need a social_github class here

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Comment by kubami9** [comment]
@akshatnitd done and squashed 😄

**Review by akshatnitd — APPROVED** [review]

**Comment by akshatnitd** [comment]
@kubami9 Great work :)

**Comment by kubami9** [comment]
Thank you! :D

**Pull request merged**